Tasks/Borohova/Task4/Task4.py

```diff
@@ -0,0 +1,37 @@
+while True:
+    h = (input("Введите кол-во символов:"))
+
+    if h.isdigit():
+        h = int(h)
+        if h <= 35:
+            print("Введите число больше 35")
+            continue
+    else:
+        print("Неверный формат ввода, введите число")
+        continue
+
+    break
+# print(h)
+
+list = []
+
+with open("text.txt", 'r') as f:
+    for l in f:
+        f1 = l.split()
+        for i in f1:
+            list.append(i)
```

Comment:
reading the whole file in memory is not optimal approach, since it can be very big